docs/AI-for-security/api/bulk-actions-anonymization-fields-api.asciidoc

```diff
@@ -0,0 +1,233 @@
+         "anonymized":false
+      },
+      {
+         "field":"agent.name",
+         "allowed":true,
+         "anonymized":true
+      }
+   ],
+}
+
+--------------------------------------------------
+
+[discrete]
+===== Response code
+
+`200`::
+    Indicates a successful call.
+
+[discrete]
+===== Response payload
+
+JSON object containing the action's outcome:
+
+- `attributes.summary.total`: Total number of anonymization fields matching the bulk action
+- `attributes.summary.succeeded`: Number of successful outcomes (number of anonymization fields that were created, deleted, or updated)
+- `attributes.summary.failed`: Number of failed outcomes
+- `attributes.summary.skipped`: Number of anonymization fields that were skipped due to various reasons (explained below)
+- `attributes.results.created`: Anonymization field objects that were created during the action's execution
+- `attributes.results.updated`: Anonymization field objects that were updated during the action's execution
+- `attributes.results.deleted`: Anonymization field objects that were deleted during the action's execution
+- `attributes.results.skipped`: Anonymization field that were skipped during the action's execution
+
+An anonymization field can only be `skipped` when the bulk action to be performed on it results in nothing being done. For example, if the `update` action is used to add a filed to a prompt that already has that value. Objects returned in `attributes.results.skipped` will only include prompts' `id`, `name`, and `skip_reason`.
+
+[source,json]
+--------------------------------------------------
+{
+    "success": true,
+    "anonymization_fields_count": 2,
+    "attributes": {
+        "results": {
+            "updated": [],
+            "created": [
+                {
+                    "timestamp": "2024-08-13T20:54:23.125Z",
+                    "createdAt": "2024-08-13T20:54:23.125Z",
+                    "field": "agent.type",
+                    "allowed": true,
+                    "anonymized": false,
+                    "updatedAt": "2024-08-13T20:54:23.125Z",
+                    "id": "cB2FTZEBYaDeA-NhPHqW"
+                },
+                {
+                    "timestamp": "2024-08-13T20:54:23.125Z",
+                    "createdAt": "2024-08-13T20:54:23.125Z",
+                    "field": "agent.name",
+                    "allowed": true,
+                    "anonymized": true,
+                    "updatedAt": "2024-08-13T20:54:23.125Z",
+                    "id": "cR2FTZEBYaDeA-NhPHqW"
+                }
+            ],
+            "deleted": [],
+            "skipped": []
+        },
+        "summary": {
+            "failed": 0,
+            "succeeded": 2,
+            "skipped": 0,
+            "total": 2
+        }
+    }
+}
+--------------------------------------------------
+
+*Example 2, Partial failure*
+
+The following request adds tags `tag-1` and `tag-2` to the rules that have the IDs sent in the payload:
```

Comment:
Not sure what rules have to do with this, and I'm not sure the example does what this line says (looks like it deletes an anonymization field entry and updates two others). Please confirm, thanks!